.travis.yml

```diff
@@ -102,20 +105,12 @@ jobs:
         - *env_osx_openssl
         - CACHE_NAME="osx.mojave"
 
-    - name: "Ubuntu 14.04 - Trusty"
-      <<: *linux_setup
-      dist: trusty
-      env:
-        - CACHE_NAME="linux.trusty"
-        - SKIP_PYTHON37_TESTS=true
-      before_install:
-        - pyenv global 2.7.14 3.6.3
-
     - name: "Ubuntu 16.04 - Xenial"
       <<: *linux_setup
       dist: xenial
       env:
         - CACHE_NAME="linux.xenial"
+        - SKIP_PYTHON38_TESTS=true
```

Comment:
What's the reason for 2 linux shards? OSX makes sense since they ship historically broken interpreters and their is only 1 shipper, Apple. For Linux though there is a large universe of shippers and Ubuntu is a very small slice. Traditionally we have had ~0 problems with linux distro pythons as well.

A further carrot is Bionic alread supports 2.7, 3.6, 3.7 & 3.8 - no need for pyenv shenanigans: https://docs.travis-ci.com/user/reference/bionic/#python-support

Reply:
That's fine with me to drop Xenial, but I'm not sure the motivation. The reason I was keeping it is is that there's little cost for us to support it in CI and it slightly improves our coverage.

For example, in the Venv PR, it looks like Trusty succeeded but for some reason Xenial and Bionic both failed (unclear why). It doesn't slow down our CI much and only adds a couple lines of config to have Xenial.

Reply:
The motivation is to drop pyenv and not pretend we cover linux. Its true 2 is more then 1, but its << 100s.

Reply:
> It doesn't slow down our CI much and only adds a couple lines of config to have Xenial.

Power consumption is a real cost we all regularaly ignore since it feels free. it's not. That's probbly the best argument. This is wasteful for no real benefit.